vhdbuilder/packer/components.json

```diff
@@ -45,7 +45,8 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "v0.0.4",
-        "v0.0.9"
+        "v0.0.9",
```

Comment:
maybe keep these in lexicographical order?

Reply:
updated